Library/Formula/appscale-tools.rb

```diff
@@ -16,62 +16,65 @@ class AppscaleTools < Formula
-    sha1 "1a2d5df8b31124573efb9598ec6d54767f3c4cd4"
+    sha256 "c36c938a872e5ff494938b33b14aaa156cb439ec67548fcab3535bb78b0846e8"
   end
 
   resource "boto" do
-    url "https://pypi.python.org/packages/source/b/boto/boto-2.9.9.tar.gz"
-    sha1 "8f8bf47a9660d6886a73c65cd00243219b5f04c4"
+    url "https://pypi.python.org/packages/source/b/boto/boto-2.38.0.tar.gz"
+    sha256 "d9083f91e21df850c813b38358dc83df16d7f253180a1344ecfedce24213ecf2"
   end
 
   resource "argparse" do
-    url "https://argparse.googlecode.com/files/argparse-1.2.1.tar.gz"
-    sha1 "caadac82aa2576d6b445058c1fcf6ef0b14dbaa1"
+    url "https://pypi.python.org/packages/source/a/argparse/argparse-1.3.0.tar.gz"
+    sha256 "b3a79a23d37b5a02faa550b92cbbbebeb4aa1d77e649c3eb39c19abf5262da04"
   end
 
   resource "oauth2client" do
-    url "https://pypi.python.org/packages/source/o/oauth2client/oauth2client-1.2.tar.gz"
-    sha1 "d38bf2d5392e4e4eea66b039a408079f6fe008d0"
+    url "https://pypi.python.org/packages/source/o/oauth2client/oauth2client-1.4.7.tar.gz"
+    sha256 "b63550a242ea25ec027a261c2f5667d52de268f52c3f0c6ce60850ad45492031"
   end
 
   resource "google-api-python-client" do
-    url "https://pypi.python.org/packages/source/g/google-api-python-client/google-api-python-client-1.2.tar.gz"
-    sha1 "31ddb6e125b0683d29493c9f486d48a4f63c913b"
+    url "https://pypi.python.org/packages/source/g/google-api-python-client/google-api-python-client-1.4.0.tar.gz"
+    sha256 "695c046789540db5ce4deefe3836e78a2cc002d5e10a41b936c6907f4ec9c96c"
   end
 
   resource "httplib2" do
-    url "https://pypi.python.org/packages/source/h/httplib2/httplib2-0.9.tar.gz"
-    sha1 "1b9774a81136a222f02e711d81efb775dc87b70e"
+    url "https://pypi.python.org/packages/source/h/httplib2/httplib2-0.9.1.tar.gz"
+    sha256 "bc6339919a5235b9d1aaee011ca5464184098f0c47c9098001f91c97176583f5"
   end
 
   resource "python-gflags" do
     url "https://pypi.python.org/packages/source/p/python-gflags/python-gflags-2.0.tar.gz"
-    sha1 "1529a1102da2fc671f2a9a5e387ebabd1ceacbbf"
+    sha256 "0dff6360423f3ec08cbe3bfaf37b339461a54a21d13be0dd5d9c9999ce531078"
   end
 
   def install
-    ENV.prepend_create_path "PYTHONPATH", libexec+"lib/python2.7/site-packages"
-
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
     resources.each do |r|
-      r.stage { system "python", "setup.py", "install", "--prefix=#{libexec}" }
+      r.stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
     end
 
-    inreplace Dir["bin/appscale*"], /^  lib = os.*/, "  lib = '#{libexec}'"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
 
-    prefix.install "bin", "templates", "LICENSE", "README.md"
-    libexec.install Dir["lib/*"]
+    system "python", "./setup.py", "build"
+    system "python", "./setup.py", "install", "--prefix=#{libexec}"
```

Comment:
Oops, didn't even see it there. Yep, just the one; setup_install_args provides arguments to python that invoke `setup.py install` with a wrapper shim and a bunch of flags.